### PR TITLE
Task001 and Task002 done.

### PR DESCRIPTION
**Pull request by ghost**
Some changes in Task001Test: changed divisionWithNaN, added tests: divisionWithInf, divisionWithNegInf.
Added tests <operation>NullOneArgument for cases <operation>(null, correct number).

### CODE REVIEW

**Comment by aabarmin** [comment]
Test already fixed. You should create pull request to your own branch not a master. Can you change the branch?

File: src/main/java/com/epam/university/java/core/task001/Task001Impl.java [hunk 1]
```diff
@@ -0,0 +1,31 @@
...
+    }
+
+    @Override
+    public double division(String firstNumber, String secondNumber) {
+        checkForNull(firstNumber, secondNumber);
+        return Double.parseDouble(firstNumber) / Double.parseDouble(secondNumber);
+    }
+
+    private void checkForNull(String firstNumber, String secondNumber) {
+        if (firstNumber == null || secondNumber == null) throw new IllegalArgumentException();
```
Comment (aabarmin) [inline src/main/java/com/epam/university/java/core/task001/Task001Impl.java]: It's better to add curly brackets.

File: src/main/java/com/epam/university/java/core/task002/Task002Impl.java [hunk 2]
```diff
@@ -0,0 +1,71 @@
+package com.epam.university.java.core.task002;
+
+public class Task002Impl implements Task002 {
+
+    private void checkForNull(Object... objects) {
+        if (objects == null) throw new IllegalArgumentException();
```
Comment (aabarmin) [inline src/main/java/com/epam/university/java/core/task002/Task002Impl.java]: Add curly braces please.

File: src/main/java/com/epam/university/java/core/task002/Task002Impl.java [hunk 3]
```diff
@@ -0,0 +1,71 @@
...
+    @Override
+    public boolean isEquals(String firstString, String secondString) {
+        checkForNull(firstString, secondString);
+        return firstString.equals(secondString);
+    }
+
+    @Override
+    public String left(String sourceString, int number) {
+        checkForNull(sourceString);
+        if (isBordersCorrect(sourceString, number))
```
Comment (aabarmin) [inline src/main/java/com/epam/university/java/core/task002/Task002Impl.java]: And here too

File: src/main/java/com/epam/university/java/core/task002/Task002Impl.java (hunk 2 above)
Comment (ghost) [reply src/main/java/com/epam/university/java/core/task002/Task002Impl.java]: done.

File: src/test/java/com/epam/university/java/core/task002/Task002Test.java [hunk 4]
```diff
@@ -50,7 +50,7 @@ public void leftNumberIsLessThenZero() throws Exception {
     public void leftWithNormalAmount() throws Exception {
         assertEquals("Error in left function",
                 "Hello",
-                instance.left("Hello, World", 5));
+                instance.left("Hegit push origin masterllo, World", 5));
```
Comment (aabarmin) [inline src/test/java/com/epam/university/java/core/task002/Task002Test.java]: Is it my mistake?)

File: src/main/java/com/epam/university/java/core/task002/Task002Impl.java (hunk 3 above)
Comment (ghost) [reply src/main/java/com/epam/university/java/core/task002/Task002Impl.java]: done.

File: src/test/java/com/epam/university/java/core/task002/Task002Test.java (hunk 4 above)
Comment (ghost) [reply src/test/java/com/epam/university/java/core/task002/Task002Test.java]: Nope. Corrected.

File: src/main/java/com/epam/university/java/core/task001/Task001Impl.java [hunk 5]
```diff
@@ -0,0 +1,31 @@
...
+    }
+
+    @Override
+    public double division(String firstNumber, String secondNumber) {
+        checkForNull(firstNumber, secondNumber);
+        return Double.parseDouble(firstNumber) / Double.parseDouble(secondNumber);
+    }
+
+    private void checkForNull(String firstNumber, String secondNumber) {
+        if (firstNumber == null || secondNumber == null) {throw new IllegalArgumentException();}
```
Comment (aabarmin) [inline src/main/java/com/epam/university/java/core/task001/Task001Impl.java]: May be add new line here?

**Comment by aabarmin** [comment]
Well done!

**Pull request merged**